docker/docker-swarm/docker-compose.yml

```diff
@@ -177,11 +177,11 @@ services:
       HADOOP_CONF_DIR: "/opt/soft/hadoop/etc/hadoop"
       SPARK_HOME1: "/opt/soft/spark1"
       SPARK_HOME2: "/opt/soft/spark2"
-      #PYTHON_HOME: "/opt/soft/python"
+      PYTHON_HOME: "/usr"
```

Comment:
This is not within the scope of this change and no modification is recommended.

Reply:
Done